src/sqlancer/Main.java

```diff
@@ -204,7 +204,7 @@ public FileWriter getReduceFileWriter() {
             }
             if (reduceFileWriter == null) {
                 try {
-                    reduceFileWriter = new FileWriter(reduceFile, false);
+                    reduceFileWriter = new FileWriter(reduceFile, true);
```

Comment:
I suppose we'd better use false here. Otherwise, the reduce log would keep increasing.

Reply:
The actual issue is at L495-497. I think the logger there is not for the `newGlobalState`